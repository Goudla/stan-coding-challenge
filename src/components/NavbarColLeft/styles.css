.navbar-col-left {
  flex: 1;
}

@media (max-width: 600px) {
  .navbar-col-left {
    display: none;
  }
}
