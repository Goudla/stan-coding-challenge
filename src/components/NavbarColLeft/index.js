import * as React from 'react';

// Styles
import './styles.css';

type Props = {
  children: React.Node
};

class NavbarColLeft extends React.PureComponent<Props> {
  render() {
    const { children } = this.props;

    return (
      <div className="navbar-col-left">
        {children}
      </div>
    );
  }
}

export default NavbarColLeft;
