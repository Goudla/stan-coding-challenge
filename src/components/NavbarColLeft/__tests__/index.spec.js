import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import NavbarColLeft from '../';

describe('<NavbarColLeft />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<NavbarColLeft>Test</NavbarColLeft>);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
