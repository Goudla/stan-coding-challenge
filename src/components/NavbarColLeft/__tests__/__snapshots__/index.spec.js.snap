// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarColLeft /> Should render 1`] = `
<div
  className="navbar-col-left"
>
  Test
</div>
`;
